usr/bin/torbrowser

```diff
@@ -410,14 +426,14 @@ maybe_install_tor_browser() {
          ## update-torbrowser is available.
          ## || true, in case update-torbrowser fails for some reason (no
          ## internet connection).
-         update-torbrowser --noaskstart || true
+         update-torbrowser "$tb_using_i2p" --noaskstart || true
```

Comment:
This should be `$tb_using_i2p` without quotes. Otherwise when unset would be `""` which then could throw an error.